src/main/java/de/tum/in/www1/artemis/service/TextSubmissionService.java

```diff
@@ -144,7 +144,7 @@ public TextSubmission save(TextSubmission textSubmission) {
     @Transactional(readOnly = true)
     public Optional<TextSubmission> getTextSubmissionWithoutResult(TextExercise textExercise) {
         Random r = new Random();
-        List<TextSubmission> submissionsWithoutResult = participationService.findByExerciseIdWithEagerSubmittedSubmissionsWithoutResults(textExercise.getId()).stream()
+        List<TextSubmission> submissionsWithoutResult = participationService.findByExerciseIdWithEagerSubmittedSubmissionsWithoutManualResults(textExercise.getId()).stream()
```

Comment:
I guess you need to rename the method here as well to reflect "without manual results" --> please also update the JavaDoc accordingly

Reply:
done